### DISCUSSION TITLE
CHIA portable

### DISCUSSION THREAD

**Discussion by lrdeoliveira** [Chia-Network/chia-blockchain#3217] [Ideas]
I would like to suggest, the application sizzles, in a portable system, so that you can use it on any computer, without having to carry and reinstall the wallet. CHIA portable

**Comment by fullet-tortuga** [comment]
That is good.

**Comment by AaronLaw** [comment]
Docker

**Comment by lrdeoliveira** [comment]
Chia Software for pendrive

**Comment by SKiV000** [comment]
good idea!
